src/app/feed/feed.component.spec.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  beforeEach,
+  beforeEachProviders,
+  describe,
+  expect,
+  inject,
+  it
+} from "@angular/core/testing";
+
+import { Control } from "@angular/common";
+
+declare const jasmine;
+declare const spyOn;
+
+import { FeedComponent } from "./feed.component";
+
+describe("Component: Feed", () => {
+  beforeEachProviders(() => {
+    return [
+      {provide: FeedComponent, useClass: FeedComponent},
+    ];
+  });
+
+  beforeEach(inject([ FeedComponent ], (feedComponent)  => {
+    this.feedComponent = feedComponent;
+    this.feedComponent.feed = {
+      isPublisher: false,
+      isLocalScreen: false,
+      isIgnored: false,
+      getVideoEnabled: jasmine.createSpy("feed.getVideoEnabled").and.returnValue(true),
+      getVideoSubscription: jasmine.createSpy("feed.getVideoSubscription").and.returnValue(false)
+      getPicture: jasmine.createSpy("feed.getPicture").and.returnValue(false),
+    };
+    this.feedComponent.highlighted = false;
+    this.feedComponent.highlightedByUser = false;
+
+    this.feedComponent.ngOnInit();
+  }));
+
+  describe("#thumbnailTag", () => {
+
+    it("should return placeholder if highlighted", () => {
```

Comment:
That's not important (just a note), but you can get rid of the _should_ to make the description shorter: "returns placeholder if highlighted".